docs/time/index.rst

```diff
@@ -1257,11 +1261,22 @@ Use of the |TimeDelta| object is illustrated in the few examples below::
   >>> t1 + 1 * u.hour
   <Time object: scale='utc' format='iso' value=2010-01-01 01:00:00.000>
 
+  # Human-readable multi-scale format for string representation of a time delta.
```

Comment:
Can we take the comment out of the block so there can be a proper link to `TimeDeltaQuantityStr`?

Reply:
Done.